exercises/find_test.rb

```diff
@@ -14,65 +14,64 @@ def test_find_first_seven_letter_word
 
   def test_no_waldo
     words = ["scarf", "sandcastle", "flag", "pretzel", "crow", "key"]
-    found = words.find do |word|
-      # Your code goes here
+    found = words.find(-> {"not here"}) do |word|
+      word == "waldo"
     end
-    assert_equal nil, found
+    assert_equal "not here", found
   end
 
   def test_find_waldo
-    skip
     words = ["noise", "dog", "fair", "house", "waldo", "bucket", "fish"]
-    # Your code goes here
+    found = words.find { |word| word == "waldo" }
     assert_equal "waldo", found
   end
 
   def test_cannot_find_3_letter_words
-    skip
     words = ["piglet", "porridge", "bear", "blueberry"]
-    # Your code goes here
+    found = words.find { |w| w.size == 3 }
     assert_equal nil, found
   end
 
   def test_find_13
-    skip
     numbers = [2, 13, 19, 8, 3, 27]
-    # Your code goes here
+    found = numbers.find { |n| n == 13 }
     assert_equal 13, found
   end
 
   def test_find_first_even_number
-    skip
     numbers = [3, 7, 13, 11, 10, 2, 17]
-    # Your code goes here
+    found = numbers.find { |n| n.odd? == false }
     assert_equal 10, found
   end
 
   def test_find_first_multiple_of_3
-    skip
     numbers = [2, 8, 9, 27, 24, 5]
-    # Your code goes here
+    found = numbers.find { |n| n % 3 == 0 }
     assert_equal 9, found
   end
 
   def test_find_first_word_starting_with_q
-    skip
     words = ["weirdo", "quill", "fast", "quaint", "quitter", "koala"]
-    # Your code goes here
+    #found = words.find { |word| word[0] == 'q' }
+    found = words.find do |word|
+      case word[0]
+      when 'q'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.